pagseguro/__init__.py

```diff
@@ -112,7 +113,7 @@ class PagSeguro(object):
     NONE = 3
 
     def __init__(self, email, token, data=None, config=None):
-        self.config = config or Config()
+        self.config = isinstance(config, AbstractConfig) or Config()
```

Comment:
That line will attributte "True" to self.config if it is an instance of AbstractConfig, and also it silence errors, I prefer

```python
def __init__(self, email, token, data=None, config=None):
    self.config = config or Config()
    assert isinstance(self.config, AbstractConfig), "Config should be instance of AbstractConfig"
```